sdlc_xmalab.py

```diff
@@ -174,6 +173,22 @@ def load_project(working_dir=os.getcwd()):
     with open(project['path_config_file'], 'w') as dlc_config:
         yaml.dump(dlc_yaml, dlc_config)
 
+    # Check DLC bodyparts (marker names) for config 2 if needed
+    if project['tracking_mode'] == 'per_cam':
+        with open(project['path_config_file_2'], 'r') as dlc_config:
+            dlc_config_loader = YAML()
+
+            dlc_yaml = dlc_config_loader.load(dlc_config)
+
+            if dlc_yaml['bodyparts'] == default_bodyparts:
+                dlc_yaml['bodyparts'] = get_bodyparts_from_xma(os.path.join(working_dir, 'trainingdata', trial), project['tracking_mode'], project['swapped_markers'], project['crossed_markers'])
```

Comment:
nit: line is too long, consider formatting to fit into 80 or 100 columns: https://google.github.io/styleguide/pyguide.html#32-line-length